schemas/groups/propulsion.json

```diff
@@ -82,6 +65,109 @@
       "units": "Pa"
     },
 
+    "alternatorVoltage": {
+      "description": "Alternator voltage",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "V"
+    },
+
+    "engineTime": {
+      "description": "Total running time for engine",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "s"
+    },
+
+    "engineTemperature": {
+      "description": "Engine temperature",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "K"
+    },
+
+    "coolantPressure": {
+      "description": "Coolant pressure",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "Pa"
+    },
+
+    "boostPressure": {
+      "description": "Engine boost (turbo, supercharger) pressure",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "Pa"
+    },
+
+    "transmission": {
+      "type": "object",
+      "properties": {
+        "gear": {
+          "type": "object",
+          "properties": {
+            "value": {
+              "enum": ["Forward", "Neutral", "Reverse", "Fault"]
```

Comment:
This could be added in the future but currently there is no mechanism for this info to be transmitted via NMEA2000. It is good to have an all encompassing spec but if the settings are not possible to set automatically from commercially available sensors and interfaces, it is a waste of time and I would prefer to add these as and when new sensors and interfaces become available.